src/pk/asn1/der/sequence/der_length_sequence.c

```diff
@@ -46,6 +52,9 @@ int der_length_sequence(ltc_asn1_list *list, unsigned long inlen,
           break;
        }
 
+       /* some items may be optional during import */
+       if (!list[i].used && list[i].optional) continue;
```

Comment:
is this one correct? this looks fishy to me as the `used` field is only used when decoding but `der_length_sequence_ex()` is used when encoding